project/plugins.sbt

```diff
@@ -3,10 +3,10 @@ resolvers += Resolver.url("HMRC-open-artefacts-ivy2", url("https://open.artefact
 resolvers += MavenRepository("HMRC-open-artefacts-maven2", "https://open.artefacts.tax.service.gov.uk/maven2")
 resolvers += "Typesafe Releases" at "https://repo.typesafe.com/typesafe/releases/"
 
-addSbtPlugin("uk.gov.hmrc"       % "sbt-auto-build"     % "3.21.0")
+addSbtPlugin("uk.gov.hmrc"       % "sbt-auto-build"     % "3.22.0")
 addSbtPlugin("uk.gov.hmrc"       % "sbt-distributables" % "2.5.0")
-addSbtPlugin("org.playframework" % "sbt-plugin"         % "3.0.2")
-addSbtPlugin("org.scoverage"     % "sbt-scoverage"      % "2.0.11")
+addSbtPlugin("org.playframework" % "sbt-plugin"         % "3.0.5")
+addSbtPlugin("org.scoverage"     % "sbt-scoverage"      % "2.2.1")
```

Comment:
Not too important since it got released after you started making the changes. 😂 But there's now a [2.2.2](https://github.com/scoverage/sbt-scoverage/releases/tag/v2.2.2)! Up to you if you want to change it.